packages/tools/u-boot/package.mk

```diff
@@ -18,6 +18,11 @@ PKG_NEED_UNPACK="$PROJECT_DIR/$PROJECT/bootloader"
 [ -n "$DEVICE" ] && PKG_NEED_UNPACK+=" $PROJECT_DIR/$PROJECT/devices/$DEVICE/bootloader"
 
 case "$PROJECT" in
+  Amlogic)
+    PKG_VERSION="2019.07-rc1"
```

Comment:
how big is the change? can you just backport the patches?

Reply:
Mostly USB-related fixes, HDMI support (not important) and added generic boards. Another option is to keep using `2019.04` and change defonfig used in `box` UBOOT_SYSTEM. I will amend this.